diff --git a/configure.ac b/configure.ac
index 02f5892..7e86c6d 100644
--- a/configure.ac
+++ b/configure.ac
@@ -9,7 +9,7 @@ dnl in order to create a configure script.
 # several non-alphanumeric characters, those are split off and used only for the
 # ABOUTBOXPRODUCTVERSIONSUFFIX in openoffice.lst. Why that is necessary, no idea.
 
-AC_INIT([LibreOffice],[4.5.0.0.alpha0+],[],[],[http://documentfoundation.org/])
+AC_INIT([LO_Platform],[4.5.0.0.20121223],[],[],[http://documentfoundation.org/])
 
 AC_PREREQ([2.59])
 
